specification/deviceupdate/resource-manager/Microsoft.DeviceUpdate/preview/2020-03-01-preview/deviceupdate.json

```diff
@@ -271,7 +317,13 @@
         ],
         "responses": {
           "200": {
-            "description": "Account updated successfully.",
+            "description": "Account was updated successfully.",
+            "schema": {
+              "$ref": "#/definitions/Account"
+            }
+          },
+          "201": {
```

Comment:
>201 [](start = 11, length = 3)

a patch should not return a 201, but a 202 to indicate async completion

Reply:
Usually PUT returns 201 and DELETE returns 202. In our case PATCH initiates the same flow as PUT hence 201.

Reply:
201 is Created, a PATCH cannot be used to create a resource - and hence it is not a valid status code to return

Reply:
We are behind RPaaS and currently they do not support 202 for async operation only 201 even for patch. We reached out to them and they said they will work either to introduce 202 or give guidance to accept 201 for there reviews in the future. For now here is the documentation https://armwiki.azurewebsites.net/rpaas/async.html If there are any questions please reach out to RPaaS Team.

Reply:
I confirmed with the RPaaS team that this is an issue today, so I will sign off. However they are looking at changing it so please work with them as to what they want you to do for this PR